### ISSUE TITLE
TORCH TENSOR

### ISSUE THREAD

**Issue by santirodriguezcalvo** [jmaronas/Deep-Learning-and-Machine-Learning#1]
FALTA EXPLICACIÓN DE COMO PONER EL TORCH TENSOR (torch.numpy(x)

## conver to torch tensor.    
x = 
## Tell pytorch this tensor requires grad so gradient is saved
x.requires_grad =

**Comment by jmaronas** [comment]
Thanks Santi.
